cityofaustin/atd-moped
# Show component location description in component list item

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/13774


## Testing
**URL to test:** 

you will have to test locally, since there are database changes

**Steps to test:**

Add a new non signal component. 
Add a location description in the form. After saving the component, the description will be visible 

![Screenshot 2023-09-13 at 9 42 02 AM](https://github.com/cityofaustin/atd-moped/assets/12474808/47942d63-ca4e-4712-a460-3e52814a393b)

For a signal component, the location description is updated with the signal's location. You should be able to edit it. 

Test changing the location description for existing components as well. There is a follow up issue to backfill the existing signal location descriptions https://github.com/cityofaustin/atd-data-tech/issues/13800

---
#### Ship list
